components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,257 @@
+1. Click **Submit**. The Automate instance details are saved.
+### Configuring Chef Automate from Chef Automate UI
+Follow these steps to configure the Chef Automate integration from the Chef Automate UI :
+1. Navigate to the **Settings** menu.
+1. Click the **Data Feeds** link from the **General** settings.
+1. Click the **Create Data Feed** button to create a Data Feed.
+1. Enter the following details:
+   - Name - a unique name for the integration.
+   - Data Feed URL – URL of the ServiceNow application Datafeed API ending `api/x_chef_automate/asset`.
+   - Chef Automate API token - an API token generated for Chef Automate with data-feed-service authorization.
+   - ServiceNow user - a ServiceNow user with **ITIL** and `x_chef_automate.api` roles.
+   - ServiceNow password - Password for the ServiceNow user being set.<br  />
+   Note that the Data Feed URL is the ServiceNow application, `FQDN/api/x_chef_automate/asset`. For example,  <https://venxxx.service-now.com/api/x_chef_automate/asset>.
+1. Click the **Test Data Feed** button. This will check that the values are correct and that there is connectivity between Chef Automate and the ServiceNow application. A successful test shows the **service-now automate connectivity passed** message. If there is an error, a message is displayed that will help resolve any connectivity or credentials issues. 
+1. Click **Create Data Feed**. The configuration is saved.
+   
+{{< figure src="/images/automate/snow_integration_create_data_feed.png" alt="Create Data Feed">}}
+## Navigation
+In ServiceNow, the navigation of the application is through the **Chef Automate** menu.
+{{< figure src="/images/automate/snow_integration_navigation.png" alt="Navigation">}}
+
+The **Automate Instances** module allows the user to configure the integration with Chef Automate. The main module for navigation is the **Servers** module. The **Server** module displays a list of servers in the _CMDB_ module. The Chef Automate integration augments the existing CMDB servers and inserts new servers into CMDB. The application uses the ServiceNow discovery IRE (Identification and Reconciliation Engine) when inserting or updating servers.<br  />
+In addition, the application updates the CMDB file systems and software installed tables, and adds related information on the servers form with related data from Chef Automate:
+- Client Runs
+- Attributes
+- Compliance Reports
+### Client Runs
+From a server record, the user can drill down into **Client Run** detail by clicking the name on an individual Client Run. Client run record displays related information for:
+- Client run cookbooks - cookbooks executed during the client run.
+- Client run lists - run lists executed during the client run.
+- Client run recipes - recipes executed during the client run.<br  />
+Full Chef client run details are available for each server. Client runs are also available from the **Client Runs** module.
```

Comment:
If we want the line 245 to not be part of the list, can give a line between them ?